src/utils/createI13nNode.js

```diff
@@ -51,34 +62,21 @@ module.exports = function createI13nNode (Component, defaultProps, options) {
     }
     var componentName = Component.displayName || Component.name || Component;
     var componentIsFunction = 'function' === typeof Component;
-    defaultProps = defaultProps || {};
-    options = options || {};
-
-    var I13nComponent = React.createClass({
-        displayName: options.displayName || ('I13n' + componentName),
-        mixins: [I13nMixin],
-        autobind: false,
+    defaultProps = Object.assign({}, {
+        i13nModel: null,
+        isLeafNode: false,
+        bindClickEvent: false,
+        follow: false,
+        scanLinks: null
+    }, defaultProps);
 
-        /**
-         * getDefaultProps
-         * @method getDefaultProps
-         * @return {Object} default props
-         */
-        getDefaultProps: function () {
-            return Object.assign({}, {
-                i13nModel: null,
-                isLeafNode: false,
-                bindClickEvent: false,
-                follow: false,
-                scanLinks: null
-            }, defaultProps);
-        },
+    options = options || {};
 
-        /**
-         * render
-         * @method render
-         */
-        render: function () {
+    class I13nComponent extends React.Component {
```

Comment:
yeah I get it, thanks!
for now we still need to extend React.component because we still need all these i13n functionalities in the component includes the life cycle/states, we do think about moving all the i13n function to the top and just do a delegation, then we might be able to apply this